cookbook-react/src/recipes/lists/list-cmpc/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+import styles from './styles.module.scss';
+import { data } from './constants';
+
+interface Item {
+  id: number;
+  name: string;
+}
+
+interface ListProps {
+  items?: Item[];
+  title?: string;
+  handleDelete?: (id: number) => void;
+}
+
+function List({ title, items, handleDelete }: ListProps) {
+  const [itemsList, setItems] = useState(items);
+
+  if (!itemsList?.length) {
+    return null;
+  }
+
+  const onDelete = (id: number) => {
+    const itemsfiltered: Item[] = itemsList.filter((item: Item) => item.id !== id);
```

Comment:
`filteredItems`

In english, the noun modifier is always before the noun itself, for example: `black cat`, `big house`, `modified element`, `filtered items`